inc/saiswitch.h

```diff
@@ -95,18 +95,43 @@ typedef enum _sai_packet_action_t
      * Packet action on the data plane remains unchanged.
      */
 
-    /** Copy Packet to CPU. */
+    /**
+     * @brief Packet action copy
+     *
+     * Copy Packet to CPU without interfering the original packet action in the
+     * pipeline.
+     */
     SAI_PACKET_ACTION_COPY,
 
     /** Cancel copy the packet to CPU. */
     SAI_PACKET_ACTION_COPY_CANCEL,
 
     /** Combination of Packet Actions */
 
-    /** This is a combination of SAI packet action COPY and DROP. */
+    /**
+     * @brief Packet action trap
+     *
+     * This is a combination of SAI packet action COPY and DROP:
+     * A copy of the original packet is sent to CPU port, the original
+     * packet is forcefully dropped from the pipeline.
+     */
     SAI_PACKET_ACTION_TRAP,
 
-    /** This is a combination of SAI packet action COPY and FORWARD. */
+#define SAI_PACKET_ACTION_BIFURCATE SAI_PACKET_ACTION_LOG
+
+    /**
+     * @brief Packet action bifurcate
+     *
+     * This is a combination of SAI packet action COPY and FORWARD:
+     * A copy of the original packet is sent to CPU port, the original
+     * packet is forcefully forwarded in the pipeline.
```

Comment:
What does the **forcefully forwarded** mean here. Does it mean any action drop is cancelled?

Reply:
Your understanding is correct. I'll update PR when after collecting more feedback.